@remirror/core-extensions/src/extensions/__tests__/history-extension.spec.ts

```diff
@@ -37,61 +36,3 @@ describe('commands', () => {
       });
   });
 });
-
-describe('`getState` and `getDispatch`', () => {
-  const dispatcher = jest.fn();
-  let state: EditorState;
-  const getState: () => EditorState = jest.fn(() => state);
-  const mocks = {
-    getState,
-    getDispatch: () => dispatcher,
-  };
-
-  const create = () => renderEditor({ others: [new HistoryExtension(mocks)] });
-  let { p, doc, add } = create();
-
-  beforeEach(() => {
-    ({ p, doc, add } = create());
-  });
-
-  it('overrides for undo', () => {
-    add(doc(p('<cursor>')))
-      .insertText('Text goes here')
-      .callback(content => {
-        state = content.state;
-      })
-      .actionsCallback(actions => {
-        actions.undo();
-        expect(mocks.getState).toHaveBeenCalledTimes(1);
-        expect(dispatcher).toHaveBeenCalledTimes(1);
-      });
-  });
-
-  it('overrides for redo', () => {
-    add(doc(p('<cursor>')))
-      .insertText('Text goes here')
-      .callback(content => {
-        state = content.state;
-      })
-      .actionsCallback(actions => {
-        actions.undo();
-        jest.clearAllMocks();
-        actions.redo();
-        expect(mocks.getState).toHaveBeenCalled();
-      });
-  });
-
-  it('overrides for keyboard shortcut', () => {
-    const { p, doc, add } = create();
-    add(doc(p('<cursor>')))
-      .insertText('Text goes here')
-      .shortcut('Mod-z')
-      .callback(content => {
-        state = content.state;
-      })
-      .callback(() => {
-        expect(mocks.getState).toHaveBeenCalledTimes(1);
-        expect(dispatcher).toHaveBeenCalledTimes(1);
-      });
-  });
-});
```

Comment:
These tests were for HistoryExtension getState() and dispatch() override feature which could prevent calling commands with dispatch=undefined so I had to remove them. As I mentioned in the PR description, if this feature is needed, I suggest implementing it globally for all extensions via extension manager.